_sass/blocks/_landing.scss

```diff
@@ -19,12 +19,13 @@
   h4 {
     font-size: 1rem;
     font-weight: $weight-bold;
-    margin-bottom: 0;
+    line-height: 1.375rem;
+    margin-bottom: 0.25rem;
   }
 
   h4,
   .h3 {
-    padding-top: 2rem - 0.75rem;
+    padding-top: 2rem - 0.5rem;
```

Comment:
I don't know what that syntax with the hyphens means; didn't code the original. I'll try variable